docs/guides/software-lifecycle/continuous-testing/testing-frameworks.md

```diff
@@ -3,119 +3,193 @@ title: Testing Frameworks
-            * Using [Scrub/Coverity/CodeQL](https://nasa.github.io/scrub/usage.html#supported-cots-tools-and-languages)
-            * Using [SonarQube](https://www.sonarqube.org/)
-    * For Web Applications
-        * For Service Endpoints (APIs)
-            * For RESTful endpoints
-                * Using [JMeter](https://jmeter.apache.org/)
-                * Using [Postman](https://www.postman.com/)
-                * Using [REST Assured (REST, RPC)](https://rest-assured.io/)
-                * Using [SOAPUI](https://www.soapui.org/)
-            * Using RPC/remote processing
-                * Using [Powershell](https://devblogs.microsoft.com/scripting/testing-rpc-ports-with-powershell-and-yes-its-as-much-fun-as-it-sounds/)
-                * Using [PortQry Interactive](https://docs.microsoft.com/en-us/troubleshoot/windows-server/networking/portqry-command-line-port-scanner-v2#step-2-specialized-tests)
-        * For User Interfaces (UIs)
-            * Using [Robot Framework](https://robotframework.org/)
-            * Using [Selenium variants \[Cucumber, etc.\]](https://github.com/SeleniumHQ/selenium)
-            * Using [Cypress](https://www.cypress.io/)
-            * Using [Playwright](https://github.com/microsoft/playwright)
-    * For Performance testing
-        * Using [JMeter](https://jmeter.apache.org/)
-        * Using [Robot Framework](https://robotframework.org/)
-        * Using [Cucumber Open](https://cucumber.io/tools/cucumber-open/)
-        * Using [Chaos Monkey](https://netflix.github.io/chaosmonkey/)
-    * For Mobile
-        * Using all platforms
-            * Using [Appium](https://appium.io/)
-        * Using Android
-            * Using [Selenium/Selendroid](http://selendroid.io/)
-            * Using [Espresso](https://developer.android.com/training/testing/espresso)
-        * Using iOS
-            * Using [Selenium/ios-driver](http://ios-driver.github.io/ios-driver/)
-            * Using [XCTest](https://developer.apple.com/documentation/xctest)
-    * For Reporting
-        * For unified analyses (dynamic and static)
-            * Using [SonarQube](https://www.sonarqube.org/)
-            * Using [Scrub/Unified](https://nasa.github.io/scrub/usage.html#supported-cots-tools-and-languages)
-        * For status reporting
-            * Using [Testrail API reporting](https://www.gurock.com/testrail/)
-            * Using [Jenkins](https://www.jenkins.io/) with [Code Coverage API Plugin](https://plugins.jenkins.io/code-coverage-api/)
-            * Using [Travis CI](https://travis-ci.org/) with [Code Climate](https://docs.travis-ci.com/user/code-climate/) or [Codecov](https://about.codecov.io/tool/travis-ci/)
-        * For real-time validation and stability
-            * Using [Jenkins](https://www.jenkins.io/)
-            * Using [GitHub Actions](https://github.com/features/actions)
-            * Using [Docker Container](https://docs.docker.com/get-started/)
+   * For Development APIs
```

Comment:
Please make sure _all_ the tools you mentioned in your guide appear somewhere on this list. For example:
- Ollama
- codellama, llama2
- Robot Framework
- etc.